pkg/storage/external/external_storage.pb.go

```diff
@@ -0,0 +1,555 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
💡 Should we add a CI check that `diffs` the commited auto-generated with the one that is getting generated by the proto file on CI. So this way we know that auto-generate code that we commit is up to date with what is expected?

Reply:
@geototti21 good idea, yes I think we should